tests/getip.c

```diff
@@ -0,0 +1,46 @@
+#include <unistd.h>
```

Comment:
I understand this tool could be convenient but libmodbus don't intend to provide such tools.